### PR TITLE
Feat/content and css

### PR DESCRIPTION
**Pull request by Simounet**
Basic content and styling.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,13 +1,28 @@
...
+      <ul>
+        <li>...</li>
+        <li>...</li>
+      </ul>
+      <p>You can find more explanations in <a href="https://doc.wallabag.org/">our documentation website</a>.</p>
+    </main>
+    <footer class="footer">
+      <ul class="bottom-links">
+        <li class="bottom-link-item"><a href="https://wallabag.org/">wallabag.org</a></li>
+        <li class="bottom-link-item"><a href="https://github.com/wallabag/wallabag">Github</a></li>
```
Comment (j0k3r) [inline index.html]:
```suggestion
        <li class="bottom-link-item"><a href="https://github.com/wallabag/wallabag">GitHub</a></li>
```

File: index.html [hunk 2]
```diff
@@ -1,13 +1,28 @@
...
-    <ul>
-      <li>...</li>
-      <li>...</li>
-    </ul>
-    You can find more explanations in our documentation website: https://doc.wallabag.org/
+    <main class="card">
+      <img src="img/logo-wallabag.svg" class="logo" alt="wallabag logo">
+      <h1 class="main-title">Setup needed</h1>
+      <p>Oh, it seems that your wallabag instance is not well configured. <br />
+      Please do some checks in your <code>`app/config/parameters.yml`</code>:</p>
```
Comment (j0k3r) [inline index.html]:
```suggestion
      Please do some checks in your <code>app/config/parameters.yml</code>:</p>
```

**Comment by nicosomb** [comment]
Do we keep this font family?

**Comment by Simounet** [comment]
As you want but I'm a really big fan of less is more.

**Review by j0k3r — APPROVED** [review]
Looks ok to me!

**Pull request merged**